modules/openapi-generator/src/main/resources/Java/libraries/jersey2/model.mustache

```diff
@@ -39,9 +51,16 @@ import javax.validation.Valid;
 {{#performBeanValidation}}
 import org.hibernate.validator.constraints.*;
 {{/performBeanValidation}}
+import {{invokerPackage}}.JSON;
```

Comment:
Hello @wing328 ,
Would you know why this import is required? I'm trying to generate code without the supporting files and it does not compile because of this import